README.md

```diff
@@ -4,32 +4,47 @@
 
 # cbuild: Open-CMSIS-Pack Build Invocation Utility
 
-This utility allows embedded developers to orchestrate the build of CPRJ projects using `cbuildgen`, `cpackget`, `cmake` and `ninja.`
+This utility allows embedded developers to build **CPRJ** and **csolution** projects by orchestrating the following tools:
+
+- `cbuildgen`
+- `csolution`
+- `cpackget`
+- `cmake`
+- `ninja`
 
 ## Usage
 
 ```bash
-cbuild: Build Invocation 1.3.0 (C) 2022 ARM
+cbuild: Build Invocation 1.3.0 (C) 2022 Arm Ltd. and Contributors
 
 Usage:
-  cbuild <project.cprj> [flags]
+  cbuild [command] <project.cprj|csolution.yml> [flags]
+
+Available Commands:
+  help            Help about any command
+  list-contexts   Print list of contexts in a csolution.yml
+  list-toolchains Print list of installed toolchains
```

Comment:
```suggestion
  list contexts   Print list of contexts in a csolution.yml
  list toolchains Print list of installed toolchains
```

Reply:
Addressed in one of the changes done in [Improvements](https://github.com/Open-CMSIS-Pack/cbuild/pull/29/commits/472704065a8867715b9fefc2c651f85f036e9c02)